html/elements/button.json

```
@@ -264,10 +264,10 @@
                 "version_added": null
               },
               "safari": {
-                "version_added": null
+                "version_added": true
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": true
```

Safari for iOS versions are slightly different though

```suggestion
                "version_added": "5"
```